Translated this page in Marathi

Abdulhussainjarif opened the pull request.

Abdulhussainjarif committed:
Translated this page in Marathi

spielers commented on docs/tutorial/part-six/index.md:
```diff
@@ -4,27 +4,27 @@ typora-copy-images-to: ./
 ---
 
-> This tutorial is part of a series about Gatsby’s data layer. Make sure you’ve gone through [part 4](/tutorial/part-four/) and [part 5](/tutorial/part-five/) before continuing here.
+> हे ट्यूटोरियल गॅट्सबीच्या डेटा लेयर बद्दलच्या मालिकेचा एक भाग आहे. आपण जात असल्याचे सुनिश्चित करा [भाग 4](/tutorial/part-four/) आणि [भाग 5](/tutorial/part-five/) येथे सुरू ठेवण्यापूर्वी.
```
आपण जात असल्याचे सुनिश्चित करा -> आपन हे एर्टिकल काळजीपुरवाक वाचल ही आपेक्षा

spielers commented on docs/tutorial/part-six/index.md:
```diff
@@ -4,27 +4,27 @@ typora-copy-images-to: ./
+## या ट्यूटोरियल मध्ये काय आहे?
 
-The previous tutorial showed how source plugins bring data _into_ Gatsby’s data system. In this tutorial, you'll learn how transformer plugins _transform_ the raw content brought by source plugins. The combination of source plugins and transformer plugins can handle all data sourcing and data transformation you might need when building a Gatsby site.
+मागील ट्युटोरियलमध्ये स्त्रोत प्लगइन डेटा कसा आणतात हे दर्शविले _into_ गॅटस्बीची डेटा सिस्टम या ट्यूटोरियल मध्ये, आपण ट्रान्सफॉर्मर प्लगइन कसे शिकू शकाल _transform_ स्त्रोत प्लगइनने आणलेली कच्ची सामग्री. सोर्स प्लगइन्स आणि ट्रान्सफॉर्मर प्लगइन यांचे संयोजन गॅटस्बी साइट तयार करताना आपल्याला आवश्यक असलेले सर्व डेटा सोर्सिंग आणि डेटा ट्रान्सफॉर्मेशन हाताळू शकते.
```
कसे add after this -> कसे वापरु

spielers commented on docs/tutorial/part-six/index.md:
```diff
@@ -4,27 +4,27 @@ typora-copy-images-to: ./
-_about_ files but what if you want to query data _inside_ files?
+बर्याचदा, आपल्याला स्त्रोत प्लगइनवरून मिळणार्या डेटाचे स्वरूप आपण आपली वेबसाइट तयार करण्यासाठी वापरू इच्छित नाही.
+फाइलप्रणाली स्त्रोत प्लगइन आपल्याला डेटा क्वेरी करू देते
+_about_ फायली परंतु आपण डेटा क्वेरी करू इच्छित असल्यास काय _inside_ फायली?
```
need to retranslate this

spielers commented on docs/tutorial/part-six/index.md:
```diff
@@ -4,27 +4,27 @@ typora-copy-images-to: ./
 
-To make this possible, Gatsby supports transformer plugins which take raw
-content from source plugins and _transform_ it into something more usable.
+हे शक्य करण्यासाठी, Gatsby ट्रान्सफॉर्मर प्लगइनचे समर्थन करते जे कच्चे करतात
```
जे कच्चे -> जे कच्चा डेटा आयत कर्ते

spielers commented on docs/tutorial/part-six/index.md:
```diff
@@ -169,11 +166,11 @@ export const query = graphql`
 ![frontpage](frontpage.png)
 
-But your one blog post looks a bit lonely. So let's add another one at
+परंतु आपला एक ब्लॉग पोस्ट थोडा एकटा दिसत आहे. तर दुसरा एक इन जोडा
```
इन -> remove this

spielers commented on docs/tutorial/part-six/index.md:
```diff
@@ -190,35 +187,32 @@ seem to really enjoy bananas!
 ![two-posts](two-posts.png)
 
-Which looks great! Except… the order of the posts is wrong.
+जे छान दिसते! वगळता… पोस्ट ऑर्डर चुकीचे आहे.
```
वगळता…  -> तारिही

spielers reviewed (changes requested):
Hey @Abdulhussainjarif  Thanks for your efforts 💐
Suggested some changes please fix this will get you merged 😀

Best,
spielers

Abdulhussainjarif committed:
Translated this file in Marathi

I have made all the changes which were suggested, please let me know if you need any more changes.

Abdulhussainjarif commented:
@spielers I have made all the suggested changes, please let me know if you need more changes!

Regards, 
Abdulhussain Jarif

spielers reviewed (approved):
Hey, @Abdulhussainjarif  Thanks

Best,
spielers,
